Index: app/src/main/java/com/app/tibibalance/ui/screens/habits/FeatureVisualizeHabits.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/app/tibibalance/ui/screens/habits/FeatureVisualizeHabits.kt b/app/src/main/java/com/app/tibibalance/ui/screens/habits/FeatureVisualizeHabits.kt
new file mode 100644
--- /dev/null	(date 1746407509496)
+++ b/app/src/main/java/com/app/tibibalance/ui/screens/habits/FeatureVisualizeHabits.kt	(date 1746407509496)
@@ -0,0 +1,2 @@
+package com.app.tibibalance.ui.screens.habits
+
